packages/react-components/react-progress/src/utils/clampMax.ts

```diff
@@ -0,0 +1,11 @@
+export const clampMax = (max: number): number => {
+  const internalMax = max !== undefined && max <= 0 ? 1 : max;
```

Comment:
You can remove the undefined check since the argument is never undefined:
```suggestion
  const internalMax = max <= 0 ? 1 : max;
```